pages/project/[id].js

```diff
@@ -189,29 +295,44 @@ const Textarea = styled.textarea`
   width: 80%;
   padding: 10px;
   border-radius: 7px;
-  border: 1px solid grey;
+  border: 2px solid;
+  border: none;
   font-size: 15px;
+  font-family: Arial, Helvetica, sans-serif;
   word-wrap: break-word;
   word-break: break-all;
-  border: none;
   font-size: 14px;
-  color: #494b49;
-  background-color: #e9eaec;
-  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 10%);
+  color: #a4a8a4;
+  background-color: #efefef;
+  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 20%);
+  ::placeholder {
```

Comment:
Does this work without an `&`? 🤔

Reply:
Yes it works 😅 But I'm willing to change it. Is it wrong with two colons?